code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -239,6 +239,9 @@ var/global/msg_id = 0
 		cam_under.pixel_y = 8
 		underlays += cam_under
 
+/obj/item/device/pda/say_quote(text)
```

Comment:
say is atom level now so say_quote is depreciated, use it instead (it is more powerful and eliminating say_quote helps us be more maintainable)

Reply:
did this even get used, anyway?

Reply:
It didn't, actually. I'll just scrap it.